static/js/containers/ImageUploader.js

```diff
@@ -99,6 +109,7 @@ const mapStateToProps = (state, ownProps) => {
     dialogOpen,
     processing,
     onUpdate,
+    name,
```

Comment:
Shouldn't `name` be passed into the component anyway? If you need it in `mergeAndInjectProps` you can destructure it form the third arg passed there.